umap/urls.py

```diff
@@ -183,7 +183,15 @@
     re_path(r"^user/(?P<identifier>.+)/$", views.user_maps, name="user_maps"),
     re_path(r"", include(i18n_urls)),
 )
-urlpatterns += (path("stats/", cache_page(60 * 60)(views.stats), name="stats"),)
+urlpatterns += (
```

Comment:
Given those are static files, can't we let nginx serve them ?

Reply:
We can put these routes before they hit Django if we control the instance but can we expect every deployment rely on nginx and admin will have the correct skills x rights to edit the configuration?

Reply:
I'd say it's a matter of documentation, we already have a bunch of Nginx configuration in the "Ubuntu" install process. But Can't we make that those file live under `/static/` so it automatically works ?

Reply:
From what I understand, the favicon has to be at the root of the server `/favicon.ico` https://en.wikipedia.org/wiki/Favicon

Reply:
Ok, that's annoying. I thought that adding those meta would allow us to customize the path…

In this case, should we put all those files in a `/static/favicon/` subfolder, and then document a simple rule in nginx ?

I'm ok to add those URLs/view in Django, but I'm just trying to challenge a less intrusive scenario. :)

Reply:
After discussion:

1. make a redirection to favicon.ico only + document the route to add in nginx
2. serve all others in static/favicons
3. create a view for the webmanifest to declare paths related to the static_url